pkg/webhooks/jobset_webhook_test.go

```diff
@@ -1599,7 +1599,7 @@ func TestValidateUpdate(t *testing.T) {
 			}.ToAggregate(),
 		},
 		{
-			name: "replicated job pod template can be updated for suspended jobset",
+			name: "replicated jobs updates can change replicas",
```

Comment:
Make sure there are two versions of the test: for suspended JobSet and unsuspended.

Reply:
Would that be relevant once #625 merges?

Reply:
sorry, I see what you mean.

Reply:
pushed up a change.